Executable/raw_injector.py

```diff
@@ -0,0 +1,128 @@
+import sys
+import ctypes
+import ctypes.wintypes
+
+from ctypes.wintypes import BOOL
+from ctypes.wintypes import DWORD
+from ctypes.wintypes import HANDLE
+from ctypes.wintypes import LPVOID
+from ctypes.wintypes import LPCVOID
+
+LPCSTR = LPCTSTR = ctypes.c_char_p
+LPDWORD = PDWORD = ctypes.POINTER(DWORD)
+
+class _SECURITY_ATTRIBUTES(ctypes.Structure):
+    _fields_ = [('nLength', DWORD),
+                ('lpSecurityDescriptor', LPVOID),
+                ('bInheritHandle', BOOL),]
+SECURITY_ATTRIBUTES = _SECURITY_ATTRIBUTES
+LPTHREAD_START_ROUTINE = LPVOID
+LPSECURITY_ATTRIBUTES = ctypes.POINTER(_SECURITY_ATTRIBUTES)
+OpenProcess = ctypes.windll.kernel32.OpenProcess
+OpenProcess.restype = HANDLE
+OpenProcess.argtypes = (DWORD, BOOL, DWORD)
+
+VirtualAllocEx = ctypes.windll.kernel32.VirtualAllocEx
+VirtualAllocEx.restype = LPVOID
+VirtualAllocEx.argtypes = (HANDLE, LPVOID, DWORD, DWORD, DWORD)
+
+ReadProcessMemory = ctypes.windll.kernel32.ReadProcessMemory
+ReadProcessMemory.restype = BOOL
+ReadProcessMemory.argtypes = (HANDLE, LPCVOID, LPVOID, DWORD, DWORD)
+
+WriteProcessMemory = ctypes.windll.kernel32.WriteProcessMemory
```

Comment:
Writing pure Python injector inside pywinauto repo allows importing class `RemoteMemoryBlock`, so this duplication is not needed probably.